src/sentry/south_migrations/0338_backfill_feature_adoption_first_project.py

```diff
@@ -0,0 +1,901 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        db.commit_transaction()
+        try:
+            self._forwards(orm)
+        except Exception:
+            # Explicitly resume the transaction because
+            # South is going to try and roll it back, but when
+            # it can't find one, it'll error itself, masking
+            # the actual exception being raised
+            #
+            # See https://github.com/getsentry/sentry/issues/5035
+            db.start_transaction()
+            raise
+        db.start_transaction()
+
+
+    def _forwards(self, orm):
+        from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+        Organization = orm['sentry.Organization']
+        OrganizationOnboardingTask = orm['sentry.OrganizationOnboardingTask']
+        FeatureAdoption = orm['sentry.FeatureAdoption']
+
+        queryset = Organization.objects.all()
+        for org in RangeQuerySetWrapperWithProgressBar(queryset):
+            if OrganizationOnboardingTask.objects.filter(organization=org, task=1, status=1).exists():
+                # Creating row in the sentry_featureadoption table.
+                # This will not update the redis cache, but this shouldn't affect
+                # the `.record` function, as create_or_update will simply update the row
+                # and insert it into the cache all the same
+                FeatureAdoption.objects.create_or_update(
```

Comment:
you can't use `create_or_update` in migrations because it's something we added to the orm